Makefile

```diff
@@ -12,7 +12,7 @@ format:
 test:
 	py.test --verbose --capture=no --exitfirst -n 6 --dist=loadscope tests
 
-before-commit: isort format lint test
+before-commit: isort lint test
```

Comment:
Please revert back. We should run `before-commit` in PY3 env. Otherwise, if you need somethign special on PY2, just run `make isort list test`